lib/actions/select.js

```diff
@@ -14,7 +14,18 @@ let connection;
  * @param cfg
  */
 function init(cfg) {
-    const conString = cfg.uri;
+    const conString = 'mssql://'
+      + cfg.username
```

Comment:
Username can't contain ':' right?

Reply:
The issue was fixed in the last commit by escaping special characters in username